lib/galaxy/webapps/galaxy/config_schema.yml

```diff
@@ -2149,6 +2149,15 @@ mapping:
           When using LDAP for authentication, allow administrators to pre-populate users
           using an additional form on 'Create new user'
 
+      enable_upload_from_form_button:
```

Comment:
I think this option is useful to overwrite in the final single-app view. So even when this is false, it should be visible in the single-form view. This said. Should this setting be a string "normal,single-app,none" or something similar?

Reply:
Is b42af61850ea0f5b4798c7e4f567e4c2a447083f good enough for now? We don't really have a single form view ready to go yet right? Once we finalize a lot of these other open PRs it'll be worth coming back and revisiting?

Reply:
Yes, this is great John!